README.md

```diff
@@ -44,9 +44,11 @@ This file contains a number of front-end interview questions that can be used wh
 * If you could master one technology this year, what would it be?
 * What are the differences between Long-Polling, Websockets and SSE?
 * Explain the importance of standards and standards bodies.
-* What is FOUC? How do you avoid FOUC?
+* What is Flash of Unstyled Content (FOUC)? How do you avoid FOUC?
 * Do your best to describe the process from the time you type in a website's URL to it finishing loading on your screen.
-* Explain what ARIA and screenreaders are, and how to make a website accessible.
+* What is your approach to accessiblity and developing in keyboard accessibility and visual focus, such as using foundational HTML and Javascript techniques? 
+* Explain your experience with Web Accessibility Initiative's Accessible Rich Internet Applications (WAI-ARIA): Do you know the difference between developing an aceesible website following Web Content Accessibility Guidelines (WCAG) web standards and custom developing for assistive technologies such as screenreaders with WAI-ARIA?
+* What experience do you have in developing in frameworks such as Microsoft Active Accessibility (MSAA), Microsoft UI Automation (UIA), or Java Accessibilty Bridge, or Linux Foundation's IAccessible2?
```

Comment:
>  If they do not have knowledge of these concepts, they have no
experience in addressing accessibility or usability for people with
disabilities.

I think that's a bit of a broad statement. It's like saying "If you haven't worked with jQuery you've never written JavaScript" (which is untrue of course). Developers can still work to address and enhance the experience of users with disabilities through a number of means and the most prevalent document/guideline we adhere to, from my experience, is the [Web Content Accessibility Guidelines - WCAG](http://www.w3.org/WAI/intro/wcag). 

I do have to admit and plead ignorance in dealing with these specific technologies myself. That said, I did some quick research and it looks like many of these frameworks deal with the browser vendor implementations and not specifically author's (ie. front-end developer's) experiences. I could be wrong but I couldn't find any recommendations or guidelines for how I, as a front-end developer, would better adhere or integrate with things like `Microsoft UI Automation`. I did, however, find some helpful MDN documents which note Firefox's [integration](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Implementing_MSAA_server) and [adherence](https://developer.mozilla.org/en-US/docs/Web/Accessibility/AT-APIs) with MSAA etc.. Java applications, notwithstanding, I agree with @richgilbank that these don't seem relevant. Feel free to correct me if I'm wrong.